lib/wal/sources/Models/Callback.hpp

```diff
@@ -16,13 +18,13 @@ namespace WAL
 	private:
 		int _nextID = 0;
 		//! @brief The list of functions to call.
-		std::unordered_map<int, std::function<void (Types...)>> _functions = {};
+		std::unordered_map<int, std::function<void(Types...)>> _functions = {};
 
 	public:
 		//! @brief Add a method to be called when this callback is invoked.
 		//! @param callback The list of arguments of the callback method
 		//! @return A unique ID for this callback. That can be used to remove the callback later.
-		int addCallback(std::function<void (Types...)> callback)
+		int addCallback(std::function<void(Types...)> callback)
```

Comment:
I believe we should keep the space, this allow us to keep the same syntax as functions & is the syntax used on any documentation you could find about std::function.

Reply:
I removed it because cpp lint thought it was a trailing space

Reply:
Change canceled -> https://github.com/AnonymusRaccoon/Bomberman/pull/56/commits/01ec0cbf05a1437e7924519e4ca6e8294b9867f6